docs/tutorials/simple-nodejs-app-ci-workflow.md

```diff
@@ -0,0 +1,108 @@
+---
+weight: 14
+title: Kev CI workflow example with a simple Node.js App
+---
+
+# CI workflow example with a simple Node.js app  
+
+This example walks through using Kev in a commit based CI pipeline.
+
+You will setup CircleCI to reconcile, render, build, push and deploy to a remote staging environment. 
+
+We assume that you,
+- Use [`git`](https://git-scm.com/) for source control and can push to a remote Git repo.
+- Have a [CircleCI](https://circleci.com/) account connected to your Git repo.
+- Have a [kube-config file](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) with access to a remote cluster.
+
+## Get Kev
+
+* Download [Kev](https://github.com/appvia/kev/releases/latest) & add it your `PATH`.
+
+## Prepare for CI
+
+### kube-config file
+
+We will encode our kube-config file into base64.
+
+```shell script
+cat ~/path/to/your/kube/config | base64
```

Comment:
I wouldn't advise that. Give an example of basic config here, specifically for the use with a target cluster.